definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-nonprod.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "LiveFrom": "01/01/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseStateID": "DNisRefused",
+    "UserRole": "caseworker-divorce-solicitor",
+    "CRUD": "RU"
+  }
+]
```

Comment:
Adding "U" for solicitor making it "RU". Changes added in the non-prod file

Reply:
You sure U is enough and we don't need C?

Reply:
The citizens journey uses just 'RU' also. Went for the best least permission.